docs/map.xsd

```diff
@@ -204,41 +358,92 @@
 <xs:attributeGroup name="property">
   <xs:attribute name="name" type="nameT" use="required"/>
   <xs:attribute name="value" type="xs:string" use="required"/>
+  <xs:attribute name="type" type="propertytypeT" use="optional" default="string"/>
 </xs:attributeGroup>
 
 <xs:attributeGroup name="map">
   <xs:attribute name="version" type="xs:string" use="required"/>
   <xs:attribute name="orientation" type="orientationT" use="required"/>
-  <xs:attribute name="renderorder" type="renderorderT"/>
+  <xs:attribute name="renderorder" type="renderorderT" default="right-down"/>
   <xs:attribute name="width" type="xs:nonNegativeInteger" use="required"/>
   <xs:attribute name="height" type="xs:nonNegativeInteger" use="required"/>
-  <xs:attribute name="tilewidth" type="xs:nonNegativeInteger" use="required"/>
-  <xs:attribute name="tileheight" type="xs:nonNegativeInteger" use="required"/>
+  <xs:attribute name="tilewidth" type="pixelT" use="required"/>
+  <xs:attribute name="tileheight" type="pixelT" use="required"/>
+  <xs:attribute name="hexsidelength" type="pixelT" use="optional"/>
```

Comment:
It just occurred to me that actually Tiled does not require the `hexsidelength` to be positive, so this should just be `xs:integer`.